nbs/tutorial_nbs/logging.ipynb

```diff
@@ -21,11 +21,11 @@
      "name": "stderr",
      "output_type": "stream",
      "text": [
-      "0:00:00.000142 \u001b[32;20mPROGRESS: test\u001b[0m\n",
+      "0:00:00.000142 \u001B[32;20mPROGRESS: test\u001B[0m\n",
```

Comment:
We have colored logging. I don't know why but it looks like your machine likes a different ANSI escape than mine???